app/assets/javascripts/govuk_publishing_components/lib/govspeak/youtube-link-enhancement.js

```diff
@@ -105,8 +105,10 @@
         events: {
           onReady: function (event) {
             // update iframe title attribute once video is ready
+            // this is done to let screen reader users know that they are focused on a video
+            // https://github.com/alphagov/govuk_publishing_components/pull/908#discussion_r302913995
             var videoTitle = options.title
-            event.target.f.title = videoTitle + ' (video)'
+            event.target.getIframe().title = videoTitle + ' (video)'
```

Comment:
Do we need to worry about the target not being a player object? for example, test that `getIframe` exists before accessing the title and such

Reply:
Because it's inside the ready function - the iframe would always exist from what I understand